### PR TITLE
FileStats: don't waste a line for just zero

### PR DESCRIPTION
**Pull request by HSGamer**

### CODE REVIEW

File: src/main/java/plugily/projects/villagedefense/user/data/FileStats.java [hunk 1]
```diff
@@ -117,8 +117,13 @@ private void updateStats(User user) {
...
 
     for(StatsStorage.StatisticType stat : StatsStorage.StatisticType.values()) {
-      if(stat.isPersistent()) {
-        config.set(uuid + "." + stat.getName(), user.getStat(stat));
+      if(!stat.isPersistent()) {
+        continue;
+      }
+      String path = user.getUniqueId().toString() + "." + stat.getName();
+      int value = user.getStat(stat);
+      if (value > 0 || config.contains(path)) {
```
Comment (montlikadani) [inline src/main/java/plugily/projects/villagedefense/user/data/FileStats.java:125]: I don't think the `contains` 2nd condition is required.

File: src/main/java/plugily/projects/villagedefense/user/data/FileStats.java [hunk 2]
```diff
@@ -117,8 +117,13 @@ private void updateStats(User user) {
     String uuid = user.getUniqueId().toString();
 
     for(StatsStorage.StatisticType stat : StatsStorage.StatisticType.values()) {
-      if(stat.isPersistent()) {
-        config.set(uuid + "." + stat.getName(), user.getStat(stat));
+      if(!stat.isPersistent()) {
+        continue;
+      }
+      String path = user.getUniqueId().toString() + "." + stat.getName();
```
Comment (montlikadani) [inline src/main/java/plugily/projects/villagedefense/user/data/FileStats.java:123]: `user.getUniqueId().toString()` -> `uuid`

File: src/main/java/plugily/projects/villagedefense/user/data/FileStats.java (hunk 1 above)
Comment (HSGamer) [reply src/main/java/plugily/projects/villagedefense/user/data/FileStats.java:125]: It's future-proof. I don't know if you guys will add some stats with negative values.

Comment (HSGamer) [reply src/main/java/plugily/projects/villagedefense/user/data/FileStats.java:125]: Or it's required when you have `SetLevelArgument`. They may set the level of their players back to 0.

**Pull request merged**